### ISSUE TITLE
Could you please provide a little example for CLI /C++CLR?

### ISSUE THREAD

**Issue by L-u1** [KennanChan/Revit.Async#30]
Hi!

thank you for the work done! I do believe RunAsync can work in CLI /C++CLR good as well?

Could you please provide a little piece of code example how could I launch function inside RevitTask::RunAsync?

Thank you,
Lu

**Comment by KennanChan** [comment]
Yes I think it can be used in C++/CLI, cause they share the same CLR.

Problem is that I got little experience in C++/CLI, which can be challenging to me to give u an example.

You may google something like "call .net static method in c++/cli" to get instructed.

I will give a try to create a demo. Not a promise and I will notice you when I get it done.

**Comment by L-u1** [comment]
Good morning,

thats would be great!
At the moment I will try inherit from IExternalEventHandler, but AFAIK, RunAsync is the wrapper for IExternalEventHandler.
Thank you in advance!

regards
Lu

**Comment by KennanChan** [comment]
Sorry, as my work focus has shifted from Revit to other areas, writing a C++/CLI demo for Revit.Async is not really my expertise. If you have already found a solution, I hope you can share it here. I will temporarily close this issue. If you have any questions, please feel free to reopen it for further discussion.
